library/core/src/ptr/const_ptr.rs

```diff
@@ -1916,7 +1962,7 @@ impl<T: ?Sized> PartialOrd for *const T {
 
 #[cfg(kani)]
 #[unstable(feature = "kani", issue = "none")]
-mod verify {
+pub mod verify {
```

Comment:
Curious why this is necessary?

Reply:
Please refer to the this [comment](https://github.com/model-checking/verify-rust-std/pull/188#discussion_r1862968896).